client/ayon_maya/api/lib.py

```diff
@@ -2011,6 +2011,7 @@ def apply_shaders(relationships, shadernodes, nodes):
     shading_engines = cmds.ls(shadernodes, type="objectSet", long=True)
     assert shading_engines, "Error in retrieving objectSets from reference"
 
+    tx_ref_nodes = cmds.ls(shadernodes, type="mesh", long=True)
```

Comment:
Let's rename this to `shader_mesh_nodes` so it's more accurate what it _actually_ is. Or `shader_meshes` or something that points out that these are just meshes... of the look (not of what's being assigned to.)